profile/src/main/java/rdx/works/profile/datastore/EncryptedPreferencesManager.kt

```diff
@@ -135,5 +147,7 @@ class EncryptedPreferencesManager @Inject constructor(
         private const val PROFILE_PREFERENCES_KEY = "profile_preferences_key"
         private const val RESTORED_PROFILE_CLOUD_PREFERENCES_KEY = "restored_cloud_profile_key"
         private const val RESTORED_PROFILE_FILE_PREFERENCES_KEY = "restored_file_profile_key"
+        private const val RETRY_COUNT = 3L
+        private const val RETRY_DELAY = 1500L
```

Comment:
Maybe we could decrease it a little ? In total it might be 4.5s so maybe would be worth decreasing to 500ms so in total it will be 1.5s  ?
Up to you, not a requirement just a thought